pkgs/tools/networking/wakeonlan/default.nix

```diff
@@ -2,18 +2,20 @@
 
 perlPackages.buildPerlPackage rec {
   pname = "wakeonlan";
-  version = "0.41";
+  version = "0.42";
 
   src = fetchFromGitHub {
     owner = "jpoliv";
     repo = pname;
-    rev = "wakeonlan-${version}";
-    sha256 = "0m48b39lz0yc5ckx2jx8y2p4c8npjngxl9wy86k43xgsd8mq1g3c";
+    rev = "v${version}";
+    sha256 = "sha256-zCOpp5iNrWwh2knBGWhiEyG9IPAnFRwH5jJLEVLBISM=";
   };
 
   outputs = [ "out" ];
 
   nativeBuildInputs = [ installShellFiles ];
+  # checkInputs = [ perl534Packages.TestPerlCritic perl534Packages.TestPod perl534Packages.TestPodCoverage ];
+  doCheck = false;  # Missing package for https://github.com/genio/test-spelling to run tests
```

Comment:
That is a spell checker and we don't run linter or coverage checks.

Reply:
I'm not too familiar with the project's architecture but it runs all of these tests when `make test` is ran, which is what the builder does during the check phase. I don't think there's a way to disable the spell tests specifically without patching the makefile.

Reply:
Then the makefile should be patched or we only run the command to run the tests. Linting and formatting checks are of no interest for us.